src/Symfony/Component/PasswordHasher/Tests/Hasher/MessageDigestPasswordHasherTest.php

```diff
@@ -57,4 +57,11 @@ public function testCheckPasswordLength()
 
         $this->assertFalse($hasher->verify('encoded', str_repeat('a', 5000), 'salt'));
     }
+
+    public function testUsingBracketInSaltThrows()
+    {
+        $this->expectException(\LogicException::class);
+        $hasher = new MessageDigestPasswordHasher('sha256', false, 1);
```

Comment:
this should be before `expectException` so that the constructor is not allowed to be the one throwing it.